installer/conf/logrotate.conf

```diff
@@ -15,3 +15,21 @@
     size 50k
     copytruncate
 }
+
+/var/log/azure/Microsoft.EnterpriseCloud.Monitoring.OmsAgentForLinux/watcher.log {
```

Comment:
just to confirm, if GA team changes this path, this won't cause logrotate to crash due to invalid path right?

Reply:
no, it wouldn't crash; missing is ok.

Of course, we'd update the path.